src/app/notifications/notifications.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit, Output, EventEmitter, ViewEncapsulation } from '@angular/core';
+import { Notification } from '../shared/swagger/model/notification';
+import { NotificationsQuery } from './state/notifications.query';
+import { NotificationsService } from './state/notifications.service';
+import { Observable } from 'rxjs';
+import { formatDate } from '@angular/common';
+
+interface DismissedNotification {
+  id: number;
+  expiration: Date;
+}
+
+@Component({
+  selector: 'app-notifications',
+  templateUrl: './notifications.component.html',
+  styleUrls: ['./notifications.component.css'],
+  encapsulation: ViewEncapsulation.None
```

Comment:
This was my suggestion as we were trying to figure out why you're styling wasn't working for the generated HTML, but I think you should remove it, and instead move the paragraph styling to the global css. The problem with that all of your component's CSS may "pollute" things.

@garyluu Don't know if you have a better idea on how to approach this? The issue is that the `<markdown>` component converts the markdown to a `<p>`, and because that paragraph element is not in the template, the component CSS doesn't affect it.

@garyluu

Reply:
I have followed your suggestion of moving the paragraph styling to the global scss file and removed the encapsulation line.